File: Sem1/ProjetoIntegrador/exerciciosPyton/L02 - avalliacao_continuada/L02B.py
#b)	distância de um raio distância = tempo x 340 ( 340 = velocidade do som no ar em metros por segundo)
import time

validation = True
valueDistance = 0
timeValue = 0

while(validation):
    try:
        value = float(input("\nDigite o valor tempo em segundos: ").replace(",", "."))
        timeValue = value
        validation = False
    except:
        print("\nValor digitado é invalido, digite novamente")
        time.sleep(2)

valueDistance = 340 * timeValue
print(f"\n\nEm {timeValue:.2f}seguntos, um riao percorre uma distância de {valueDistance:.2f}metros")

File: Sem1/ProjetoIntegrador/exercicios/src/tarefaII/avaliacaoContinua/L02B.java
//b)Distância de um raio: distancia = tempo X 360metros/segundo
package tarefaII.avaliacaoContinua;

import javax.swing.*;

public class L02B {
    public static void main(String[] args) {
        boolean validation = true;
        int secunds = 0;
        double distancia;
        do {
            try {
                secunds = Integer.parseInt(JOptionPane.showInputDialog("Digite o valor em segundos, para saber a distância do raio").replace(",",""));
                if (secunds > 0) {
                    validation = false;
                }else {
                    JOptionPane.showMessageDialog(null, "Valor digitado invalido, tente novamente: ");
                }
                } catch (Exception e) {
                JOptionPane.showMessageDialog(null, "Valor digitado invalido, tente novamente: ");
            }
        }while (validation);

        distancia = lightningStrikeDistance(secunds);
        JOptionPane.showMessageDialog(null,
                "\nO som viaja a 360m/s;"
                        +"\nComo o som do raio foi ouvido "+secunds+" segundos após ser visto,"
                        +"\no raio está á " + String.format("%.2f",distancia) + "metros de distâcia!");


    }
    public static double lightningStrikeDistance(int time){
        double distance = time * 360;
        return distance;
    }
}
